_layouts/hall-of-fame.html

```diff
@@ -0,0 +1,57 @@
+---
+layout: base
+---
+
+{% assign contributors = site.data['contributors'] %}
+
+
+{% include default-header.html %}
+
+<div class="container main-content">
+<section>
+<h1>Hall of Fame</h1>
+<p>A big <i>Thank You!</i> to the <strong>{{ contributors | size }}</strong> awesome contributors to the Galaxy training materials! </p>
+
+<div class="row">
+
+    {% for contributor in contributors %}
+        {% assign username = contributor[0] %}
+        <div class="col-2 hall-of-fame" id="{{ username }}">
+        {% if contributors[username].github != false %}
+        <a href="https://github.com/{{ username }}" class="thumbnail withlink">
+        {% else %}
+        <a class="thumbnail">
+        {% endif %}
+            <img src="https://avatars.githubusercontent.com/{{ username }}" alt="{{ username }}"/>
+        </a>
+        <br/>
```

Comment:
We should not have `<br>` very often.

Reply:
ah yes, I was lazy :P will try to replace these with css soon